frontend/build.yaml

```diff
@@ -15,11 +15,31 @@ targets:
           include:
             - "lib/build_config/build_config.yaml"
         enabled: True
-      df_protobuf:
-        generate_for:
-          include:
-            - "card.proto"
-        enabled: True
+      protoc_builder:
+        options:
+          # The version of the Protobuf compiler to use.
```

Comment:
:thinking: do we need all those comments? Arent they usually self explanatory/could be easily looked up in the documentation?

Reply:
I removed them